packages/manager/modules/office/src/translations/Messages_fr_FR.json

```diff
@@ -1,4 +1,13 @@
 {
   "office_title": "Microsoft 365",
-  "office_order": "Commander"
+  "office_order": "Commander",
+  "office_license_detail_user_firstname": "First Name",
+  "office_license_detail_user_lastname": "Last Name",
+  "office_license_detail_user_address": "Address",
+  "office_license_detail_user_city": "City",
+  "office_license_detail_user_display_name": "Display Name",
+  "office_license_detail_user_service_name": "Service Name",
+  "office_license_detail_user_service_type": "Service Type",
+  "office_license_detail_user_status": "Status",
+  "office_license_detail_user_zip_code": "Zip Code"
```

Comment:
This should be text FR

Reply:
Removed it and moved it to constant file